### DISCUSSION TITLE
SEB is not launching in windows 11

### DISCUSSION THREAD

**Discussion by nputhuva-rm** [SafeExamBrowser/seb-win-refactoring#958] [Q&A]
getting below error while launching SEB in windows 11 machine
![image](https://github.com/user-attachments/assets/a5eb805c-c1de-4f45-99fa-29f88b57f1a1)
 
 Please find the attached log file
[2024-08-28_14h21m59s_Runtime.log](https://github.com/user-attachments/files/16792256/2024-08-28_14h21m59s_Runtime.log)
@dbuechel  Could you please help with this?

**Comment by dbuechel** [comment]
There are already various issues and discussions related to this issue here on GitHub. From what I recall, it can be caused by a generic display driver, a missing or outdated graphics card driver or a corrupted system configuration.
